admin-dev/themes/new-theme/js/components/grid/extension/position-extension.js

```diff
@@ -31,141 +31,145 @@ const $ = window.$;
  * Class PositionExtension extends Grid with reorderable positions
  */
 export default class PositionExtension {
-    constructor() {
-        return {
-            extend: (grid) => this.extend(grid),
-        }
+  constructor() {
+    return {
+      extend: (grid) => this.extend(grid),
     }
+  }
 
-    /**
-     * Extend grid
-     *
-     * @param {Grid} grid
-     */
-    extend(grid) {
-        this.grid = grid;
-        this._addIdsToGridTableRows();
-        grid.getContainer().find('.js-grid-table').tableDnD({
-            onDragClass: 'position-row-while-drag',
-            dragHandle: '.js-drag-handle',
-            onDragStart: () => {
-                this.originalPositions = decodeURIComponent($.tableDnD.serialize());
-            },
-            onDrop: (table, row) => this._handlePositionChange(row),
-        });
-    }
-
-    /**
-     * When position is changed handle update
-     *
-     * @param {HTMLElement} row
-     *
-     * @private
-     */
-    _handlePositionChange(row) {
-        const $rowPositionContainer = $(row).find('.js-' + this.grid.getId() + '-position:first');
-        const updateUrl = $rowPositionContainer.data('update-url');
-        const method = $rowPositionContainer.data('update-method');
-        const paginationOffset = parseInt($rowPositionContainer.data('pagination-offset'));
-        const positions = this._getRowsPositions(paginationOffset);
-        const params = {
-            positions: positions,
-        };
+  /**
+   * Extend grid
+   *
+   * @param {Grid} grid
+   */
+  extend(grid) {
+    this.grid = grid;
+    this._addIdsToGridTableRows();
+    grid.getContainer().find('.js-grid-table').tableDnD({
+      onDragClass: 'position-row-while-drag',
+      dragHandle: '.js-drag-handle',
+      onDrop: (table, row) => this._handlePositionChange(row),
+    });
+    grid.getContainer().find('.js-drag-handle').on('mouseenter', (event) => {
+      $(event.target).closest('tr').addClass('hover');
+    });
+    grid.getContainer().find('.js-drag-handle').on('mouseleave', (event) => {
+      $(event.target).closest('tr').removeClass('hover');
+    });
+  }
 
-        this._updatePosition(updateUrl, params, method);
-    }
+  /**
+   * When position is changed handle update
+   *
+   * @param {HTMLElement} row
+   *
+   * @private
+   */
+  _handlePositionChange(row) {
+    const $rowPositionContainer = $(row).find('.js-' + this.grid.getId() + '-position:first');
+    const updateUrl = $rowPositionContainer.data('update-url');
+    const method = $rowPositionContainer.data('update-method');
+    const paginationOffset = parseInt($rowPositionContainer.data('pagination-offset'));
```

Comment:
always add the base when using parseInt:
`parseInt($rowPositionContainer.data('pagination-offset'), 10);`

Reply:
isn't the default base 10?